include/gpio_wrapper.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "nrf_gpio.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+namespace edge::aidan {
+
+inline bool read_gpio(uint32_t pin_number)
+{
+    bool val = nrf_gpio_pin_read(pin_number);
+    return val;
+}
+
+inline void set_gpio(uint32_t pin_number)
```

Comment:
nit: set_gpio_pin instead of set_gpio (applies to all these functions)

Reply:
done